ui/admin/setup-edit.php

```diff
@@ -146,7 +146,7 @@
     <input type="hidden" name="old_name" value="<?php echo esc_attr( $pod[ 'name' ] ); ?>" />
 
     <h2>
-        Edit Pod:
+        <?php _e( 'Edit Pod', 'pods' ); ?>:
```

Comment:
Please use `esc_html_e()` instead of `_e()` for echoing values that do not require HTML.